### PR TITLE
MET+MHT+b-tag Offline DQM 92X

### PR DESCRIPTION
**Pull request by jmduarte**
These changes depend on the PRs #19293 (MET paths) and #19119 (b-tag paths). I can rebase once these are merged.

My commits are only the final two:
https://github.com/cms-sw/cmssw/pull/19508/files/3354e5e5d53ef285be8fbc062c9f474c2d29c309..4ed4df680ee8e276db9dd1574e59ad1d8ba2ca8a

### CODE REVIEW

File: DQMOffline/Trigger/plugins/HTMonitor.cc [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+  edm::Handle<reco::PFJetCollection> jetHandle; //add a configurable jet collection & jet pt selection
+  iEvent.getByToken( jetToken_, jetHandle );
+  std::vector<reco::PFJet> jets;
+  jets.clear();
+  if ( jetHandle->size() < njets_ ) return;
+  for ( auto const & j : *jetHandle ) {
+    if ( jetSelection_( j ) ) {
+      jets.push_back(j);
+      if ( jetSelection_HT_(j)) ht += j.pt();
```
Comment (mtosi) [inline DQMOffline/Trigger/plugins/HTMonitor.cc]: uff, please drop this line !

**Comment by mtosi** [comment]
tracked by https://github.com/cms-sw/cmssw/issues/19142

**Comment by fwyzard** [comment]
@cmsbuild, please test with #19119 #19293

File: DQMOffline/Trigger/plugins/HTMonitor.cc (hunk 1 above)
Comment (jmduarte) [inline DQMOffline/Trigger/plugins/HTMonitor.cc]: @mtosi ok I can, but for your information, this code is present in the PR #19293 

https://github.com/rappoccio/cmssw/blob/0f13299cb02c84e6319df259849563d63c1d4d77/DQMOffline/Trigger/plugins/HTMonitor.cc#L164-L167

I just did a merge-topic and then added a few configuration changes on top of #19119 and #19293.

See my commit here: https://github.com/cms-sw/cmssw/pull/19508/commits/83c51a305812b955bcbc232c9dd3abd7224dbb3e

**Comment by jmduarte** [comment]
I rearranged the folders so the DQM histograms will live in HLT/Higgs/

The last two commits are the only ones on top of #19119 #19293:
https://github.com/cms-sw/cmssw/pull/19508/files/3354e5e5d53ef285be8fbc062c9f474c2d29c309..4ed4df680ee8e276db9dd1574e59ad1d8ba2ca8a

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by dmitrijus** [comment]
+1

**Comment by mtosi** [comment]
do we have the analogous PR for the master (93x) ?

**Comment by jmduarte** [comment]
93X version of PR is here: #19596

**Comment by mtosi** [comment]
in the silvio's google doc I find

HLT_PFMET100_PFMHT100_IDTight_BTagCaloCSV_p7_v*
HLT_PFMET110_PFMHT110_IDTight_BTagCaloCSV_p7_v*
HLT_PFMET120_PFMHT120_IDTight_BTagCaloCSV_p7_v*
HLT_PFMET130_PFMHT130_IDTight_BTagCaloCSV_p7_v*
HLT_PFMET140_PFMHT140_IDTight_BTagCaloCSV_p7_v*
!!!!

can we converge to the name of those trigger paths, please ?

**Comment by jmduarte** [comment]
Yes, sorry, I renamed them as requested by TSG rather late... I asked @silviodonato to rename them in the google doc and the master table in the JIRA ticket:

https://its.cern.ch/jira/browse/CMSHLT-1391

The DQM PR here always refer to the p05 (mistag efficiency) names.

Thanks,

Javier

**Comment by fwyzard** [comment]
> I renamed them as requested by TSG rather late...

The "request" was actually agreed between TSG and b-tagging POG one year ago.
Then they apparently forgot.

.A

**Comment by fwyzard** [comment]
Hi,
given the amount of stuff that needs to be backported, I plan to do it in a single PR as soon as the DQM for the HLT menu v2.1 is fully integrated in 9.3.x.
If the 9.3.x equivalent of this PR has already been merged, or is listed as "Open" at #19412, can you close this PR ?

**Comment by jmduarte** [comment]
Hi @fwyzard, I assume you mean #19142. 

In that PR, I do see the 93X version of this PR #19596 with the status "Open" (only ORP pending).

Given that, I'll close this PR.

Thanks,

Javier

**Pull request closed without merging**